.github/workflows/run-eva-submission-tests.yml

```
@@ -0,0 +1,31 @@
+name: Run Tests from EVA Sub Cli
+
+on:
+  repository_dispatch:
+    types: [trigger-eva-submission-cli-tests]
```

```suggestion
    types: [trigger-eva-submission-tests]
```
Assuming this will be triggered from eva-submission repo only, I think this name would be more clear.

Will setting up the trigger event from eva-submission be a separate PR?